src/types/components/enhanced-table/enhancedTable.interface.ts

```diff
@@ -0,0 +1,44 @@
+import { ChangeEvent, MouseEventHandler } from 'react'
+import { Sort } from '~/types/common/common.index'
+
+export interface TableColumn {

```

Comment:
can it have label?

Reply:
sorry, but i didnt get it, do you mean to remove table and just stay Column without "Table" and the same to other interfaces?